### PR TITLE
Initial Catkinization

### LINKED ISSUE #14: vetex_support is not a catkin package
May not be an actual issue, but at least on my machine this caused `ResourceNotFound` exceptions trying to launch `euler_navigation_demo stage.launch`.

### PR DESCRIPTION
**Pull request by DLu**
Fixes #14

### CODE REVIEW

File: vetex_support/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(vetex_support)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
```
Comment (gavanderhoorn) [inline vetex_support/CMakeLists.txt:7]: Would we perhaps want to add `INSTALL` targets here?

Something like:

``` cmake
foreach(dir launch meshes urdf)
  install(
    DIRECTORY ${dir}/
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
endforeach()
```

File: vetex_support/package.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<package>
+  <name>vetex_support</name>
+  <version>0.0.0</version>
+  <description>The vetex_support package</description>
+  <maintainer email="paul.hvass@swri.org">Paul Hvass</maintainer>
+  <author>Paul Hvass</author>
+  <license>BSD</license>
+  <buildtool_depend>catkin</buildtool_depend>
```
Comment (gavanderhoorn) [inline vetex_support/package.xml:9]: Should this also list a `run_depend` on `xacro`? The `display.launch` file uses it.

Also: `joint_state_publisher`, `robot_state_publisher` and `rviz`.

**Comment by gavanderhoorn** [comment]
@DLu: can probably ignore my comments. #16 addresses them (apart from the `INSTALL` targets).

File: vetex_support/CMakeLists.txt (hunk 1 above)
Comment (DLu) [inline vetex_support/CMakeLists.txt:7]: Fixed.

**Comment by DLu** [comment]
With the added install targets, is this ready to merge? @gavanderhoorn

**Comment by gavanderhoorn** [comment]
@DLu: yes, go ahead. `run_depends` are added in #16.

**Comment by shaun-edwards** [comment]
Sounds like a +1, @gavanderhoorn, thanks for the review.

**Pull request merged**